core/protocols/aws-xml-protocol/src/main/java/software/amazon/awssdk/protocols/xml/internal/unmarshall/HeaderUnmarshaller.java

```diff
@@ -69,9 +73,17 @@ private SimpleHeaderUnmarshaller(StringToValueConverter.StringToValue<T> stringT
 
         @Override
         public T unmarshall(XmlUnmarshallerContext context, List<XmlElement> content, SdkField<T> field) {
-            return context.response().firstMatchingHeader(field.locationName())
-                          .map(s -> stringToValue.convert(s, field))
-                          .orElse(null);
+            try {
+                return context.response().firstMatchingHeader(field.locationName())
+                              .map(s -> stringToValue.convert(s, field))
+                              .orElse(null);
+            } catch (RuntimeException e) {
+                log.warn("Exception found while parsing response header {}", e.getMessage());
```

Comment:
I wonder if this might end up being too noisy for users. Maybe not, I'm not sure... Could there be use cases where their logs get swarmed with this warning? Like a user accessing the same object which has this bad header a lot?

Reply:
I do not think this is a very common use case. Moreover, this is an explicit SEP requirement 
```
The SDKs MUST log a warning message and return null for that field.
```
https://code.amazon.com/packages/AwsDrSeps/blobs/main/--/seps/accepted/shared/s3-expires-shape-customization.md